sentry/src/main/java/io/sentry/hints/Hints.java

```diff
@@ -0,0 +1,29 @@
+package io.sentry.hints;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class Hints {
+
+  private final @NotNull Map<String, Object> internalStorage = new HashMap<String, Object>();
+
+  public void set(@NotNull String hintType, @Nullable Object hint) {
```

Comment:
`m`: I think we can remove hint in the naming. It's kind of redundant.
```suggestion
  public void set(@NotNull String type, @Nullable Object hint) {
```

Reply:
@adinauer, did you resolve this by accident, or did you forget to mention why you don't think my change makes sense?

Reply:
Ah sorry forgot to mention that I renamed it to `name` in #2046